tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/int_problem_results.yaml

```diff
@@ -235,7 +236,7 @@ metrics:
   d3format: null
   description: null
   expression: "  ROUND(\r\n    (SUM(CASE WHEN not success THEN attempts ELSE 0 END))\
-    \ /\r\n    SUM(attempts), 2\r\n  )*100"
```

Comment:
Currently these are still necessary, the correct/incorrect % values are showing up as `0.97` etc